cmake/dts.cmake

```
@@ -247,6 +252,20 @@ if(SUPPORTS_DTS)
     message(STATUS "Generated device_extern.h: ${DEVICE_EXTERN_H}")
   endif()
 
+  execute_process(
+    COMMAND ${PYTHON_EXECUTABLE} ${DTS_CMAKE_SCRIPT}
+    --edt-pickle ${EDT_PICKLE}
+    --cmake-out ${DTS_CMAKE}
+    WORKING_DIRECTORY ${PROJECT_BINARY_DIR}
+    RESULT_VARIABLE ret
+    )
+  if(NOT "${ret}" STREQUAL "0")
+    message(FATAL_ERROR "dts_cmake.py failed with return code: ${ret}")
+  else()
+    message(STATUS "Generated dts.cmake: ${DTS_CMAKE}")
```

```suggestion
    message(STATUS "Including generated dts.cmake file: ${DTS_CMAKE}")
```

This makes it clearer to the user what its purpose is IMO.

---

fixed